### ISSUE TITLE
Possible to output NM aux tag?

### ISSUE THREAD

**Issue by wwood** [rcedgar/urmap#5]
Hi,

I'm the maintainer of [CoverM](https://github.com/wwood/CoverM), which thresholds SAM alignments using the NM edit distance, used by BWA and minimap2. Is it possible to add this as a tag in the output sam for urmap somehow?

Thanks, ben

**Comment by rcedgar** [comment]
Good idea, but it's going to be a while before I can get to this unfortunately.

**Comment by jianshu93** [comment]
This will be a great improvement in terms of mapping speed for most metagenomic analysis (abundance and coerage calculation et.al). I would say urmap will be very popular in the future as Metagenomic datasets is becoming larger and larger.

**Comment by jianshu93** [comment]
Any updates on this thread? This is very useful for any downstream coverage depth of reference.

Thanks,

Jianshu

**Comment by rcedgar** [comment]
sorry still backlogged with other projects, will be a few weeks minimum.
